fe/fe-core/src/main/java/com/starrocks/analysis/SlotRef.java

```diff
@@ -107,11 +107,12 @@ public SlotRef(QualifiedName qualifiedName) {
             this.col = parts.get(3);
             this.label = parts.get(3);
         } else {
+            // If parts.size() > 4, it must refer to a struct subfield name, so we set SlotRef's TableName null value,
+            // set col, label a qualified name here[Of course it's a wrong value].
+            // Correct value will be parsed in Analyzer according context.
```

Comment:
if it's wrong, why not just give it empty string.

Reply:
We need to return the error msg to the user if you set col = "", the error msg will be empty too.